src/src/components/navigation/Profile.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+<div class="profile-main-container">
+  <div class="profile-brand">
+    <img
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 4 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
      <img
```